clients/include/testing_gbmv_strided_batched.hpp

```diff
@@ -97,47 +98,62 @@ hipblasStatus_t testing_gbmvStridedBatched(const Arguments& argus)
     hipblas_init<T>(hy, 1, y_els, incy, stride_y, batch_count);
 
     // copy vector is easy in STL; hz = hy: save a copy in hz which will be output of CPU BLAS
-    hz = hy;
+    hy_cpu = hy;
 
     // copy data from CPU to device
-    hipMemcpy(dA, hA.data(), sizeof(T) * A_size, hipMemcpyHostToDevice);
-    hipMemcpy(dx, hx.data(), sizeof(T) * X_size, hipMemcpyHostToDevice);
-    hipMemcpy(dy, hy.data(), sizeof(T) * Y_size, hipMemcpyHostToDevice);
+    CHECK_HIP_ERROR(hipMemcpy(dA, hA.data(), sizeof(T) * A_size, hipMemcpyHostToDevice));
```

Comment:
Another comment not for this PR, but we should have added hA.dataSize() for all these types of transfers.